last_seen.py

```diff
@@ -0,0 +1,28 @@
+from brutal.core.plugin import BotPlugin, cmd, event
+from time import strftime, gmtime
+
+class LastSeen(BotPlugin):
+    def setup(self, *args, **kwargs):
+        self.seen = self.open_storage('last_seen')
+
+    @event
+    def seen_evt(self, event):
+        events = ['quit', 'part', 'kick', 'join', 'rename', 'topic']
+        if event.event_type in events:
+            nick = event.meta['nick']
+            time = strftime("%d.%m.%Y", gmtime())
```

Comment:
we probably want to know hours/minutes, don't we?

You could save the time of last activity, and then in last_seen cmd return something like "2 hours ago"